.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -8,7 +8,7 @@ about: Suggest an idea for this project
 <!-- A clear and concise description of what the problem is. -->
```

Comment:
This should probably also be clearer that we want something concrete. Maybe "A clear and concise description of the problem that this feature is intended to solve. This should include reference to actual use cases and workflows."?

Or maybe something more informal, like "What are you trying to do, that you are unable to achieve with pip as it currently stands?"

Reply:
I like the latter. :)